samples/spm/prj.conf

```diff
@@ -16,3 +16,8 @@ CONFIG_MAIN_STACK_SIZE=4096
 # being built by a parent image. Hence we set it here to ensure that SPM
 # cleans up the core during boot.
 CONFIG_INIT_ARCH_HW_AT_BOOT=y
+
+# When enabled, spm can print to UARTE0 even after switching to non-secure, e.g.
+# from secure fault handler. zephyr,console chosen must be uncommented in the
+# overlay file.
+#CONFIG_SPM_SHARE_CONSOLE_UART=y
```

Comment:
wouldn't it be better to place this in a dedicated Kconfig fragment, so that users don't have to modify `prj.conf` to enable this, but can enable this through the use of `OVERLAY_CONFIG` ?

That is, make a `uart_share.config.conf`:
```
# This Kconfig fragment allows spm to print to UARTE0 even after switching to non-secure, e.g.
# from secure fault handler.
# To enable when building spm: `west build ... -- -DOVERLAY_CONFIG=uart_share.config.conf
# To enable when building spm in multiimage context: `west build ... -- -spm_DOVERLAY_CONFIG=uart_share.config.conf
CONFIG_SPM_SHARE_CONSOLE_UART=y
```